cmd/sriov-network-config-daemon/service.go

```diff
@@ -163,12 +164,24 @@ func phasePre(setupLog logr.Logger, conf *systemd.SriovConfig, hostHelpers helpe
 	hostHelpers.TryEnableTun()
 	hostHelpers.TryEnableVhostNet()
 
-	return callPlugin(setupLog, PhasePre, conf, hostHelpers)
+	// Add retry here as some time the kernel is not fully loaded, and we see issues like
+	// unbindDriver(): failed to unbind driver "error": "open /sys/bus/pci/drivers/igbvf/unbind: permission denied"
+	i := 0
+	for i < 5 {
```

Comment:
can you move this change to a separate PR?
I think we should review discuss it separately.

Reply:
Was the problem that is described in the comment ever seen with this patch?https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/808 I think it can implicitly fix the issue.

Reply:
I will try again but for sure that is not part of this PR.

I am working on another PR to add a systemd mode functional test lane, that is how I catch it.